### PR TITLE
More shepherd comments

### PR DESCRIPTION
**Pull request by smyslov**

### CODE REVIEW

**Comment by smyslov** [comment]
XML is also fixed so that datatracker must accept it.

File: draft-ietf-ipsecme-rfc8229bis.xml [hunk 1]
```diff
@@ -1392,7 +1393,7 @@
               <t>Once the client receives a response on the
               TCP-encapsulated connection, it immediately start new INFORMATIONAL
```
Comment (tfpauly) [inline draft-ietf-ipsecme-rfc8229bis.xml]:
```suggestion
              TCP-encapsulated connection, it immediately starts a new INFORMATIONAL
```

File: draft-ietf-ipsecme-rfc8229bis.xml [hunk 2]
```diff
@@ -1392,7 +1393,7 @@
...
               exchange with UPDATE_SA_ADDRESSES notify payload and recalculated
```
Comment (tfpauly) [inline draft-ietf-ipsecme-rfc8229bis.xml]:
```suggestion
              exchange with an UPDATE_SA_ADDRESSES notify payload and a recalculated 
```

File: draft-ietf-ipsecme-rfc8229bis.xml [hunk 3]
```diff
@@ -1392,7 +1393,7 @@
               <t>Once the client receives a response on the
               TCP-encapsulated connection, it immediately start new INFORMATIONAL
               exchange with UPDATE_SA_ADDRESSES notify payload and recalculated 
-              NAT_DETECTION_* notify payloads to get correct information about the precense 
+              NAT_DETECTION_SOURCE_IP notify payload to get correct information about the precense
```
Comment (tfpauly) [inline draft-ietf-ipsecme-rfc8229bis.xml]:
```suggestion
              NAT_DETECTION_SOURCE_IP notify payload in order to get correct information about the presence 
```

File: draft-ietf-ipsecme-rfc8229bis.xml [hunk 4]
```diff
@@ -1392,7 +1393,7 @@
               <t>Once the client receives a response on the
               TCP-encapsulated connection, it immediately start new INFORMATIONAL
               exchange with UPDATE_SA_ADDRESSES notify payload and recalculated 
-              NAT_DETECTION_* notify payloads to get correct information about the precense 
+              NAT_DETECTION_SOURCE_IP notify payload to get correct information about the precense 
               of NAT.
```
Comment (tfpauly) [inline draft-ietf-ipsecme-rfc8229bis.xml]:
```suggestion
              of NATs.
```

**Review by tfpauly — APPROVED** [review]
Looks good, just grammar nits

**Pull request merged**